help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,158 @@
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL URL]**: Pega Customer Decision Hub Service Rest Segment membership Endpoint URL. The Service endpoint URL is available by opening Service Rest rule type of Adobe Segment Membership Component enabled on your Pega instance.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+See [Activate audience data to streaming profile export destinations](../../ui/activate-streaming-profile-destinations.md) for instructions on activating audience segments to this destination.
+
+### Destination attributes {#attributes}
+
+In the [[!UICONTROL Select attributes]](../../ui/activate-streaming-profile-destinations.md#select-attributes) step, Adobe recommends that you select a unique identifier from your [union schema](../../../profile/home.md#profile-fragments-and-union-schemas). Select the unique identifier and any other XDM fields that you want to export to the destination.
+
+### Mapping example: activating profile updates in [!DNL Pega Customer Decision Hub] 
+
+Below is an example of correct identity mapping when activating profile updates in [!DNL Pega Customer Decision Hub].
+
+Selecting source fields:
+
+* Select an Identifier (For eg: CustomerID) as source identity that uniquely identifies a profile in Adobe Experience Platform and  Pega Customer Decision Hub.
+* Select XDM source profile attribute changes that need to be updated in Pega Customer Decision Hub.
+
+Selecting target fields:
+
+* Select the `CustomerID` namespace as target identity.
+* Select destination profile attribute names that need to be mapped to corresponding XDM source profile attributes.
+
+![Identity mapping](../../assets/catalog/http/pega/pega-source-destination-mapping.png)
+
+## Exported data / Validate data export {#exported-data}
+Your exported [!DNL Experience Platform] data lands in your [!DNL HTTP] destination in JSON format. For example, the export snippet below contains a profile that is qualified for a certain segment, is a member of another two segments, and exited another segment. The export also includes the profile attribute first name (*FirstName*), last name (*LastName*), date of birth (*BirthDate*), and Customer Identfier (*CustomerID*) for this profile.
```

Comment:
Comment applies to rows 111 - 140 (suggestion to remove these lines):

I don't think that this applies to the Pega destination. This is just an example that we use for the HTTP API destination and other destinations where you can inspect the JSON in the destination platform. This does not look to be the case in Pega, and in Pega the relevant information is in the Pega UI.

Reply:
ok. makes sense. Removed